src/components/aside/Aside.jsx

```diff
@@ -12,19 +12,20 @@ export const Aside = () => {
     pathname !== "/signup" ? (
     <aside className="aside">
       <NavMenu>
-        {asideData.map(({ id, name, path }) => {
+        {asideData.map(({ id, name, path,icon }) => {
           return (
-            <NavItem key={id}>
-              <NavLinks
-                to={path}
-               exact activeStyle={{
-                color: 'red',
-              }}
+            <NavLinks key={id}  to={path}   exact activeStyle={{
+              color: 'red ',
+            }}>
+                <IconWrapper>{icon}</IconWrapper>
+              <NavItem 
+             
                 style={{ color: getAsideColor(theme) }}
```

Comment:
Try avoiding inline css if possible

Reply:
Okay, Here I am using that so that I can get the color according to theme state